readme: Notice for Linux users (#61)
tailscale-dev/vscode-tailscale: README.md

  <a title="Learn more about Tailscale" href="https://tailscale.com"><img width="476px" src="https://github.com/tailscale-dev/tailscale-dev/assets/40265/113339dc-fea6-4ccf-a284-22ecee62c7c9" alt="Tailscale Logo" /></a>
</p>

> > Notice for Linux users: We're currently working to resolve an issue preventing this extension from being used on Linux. However, you're in luck and can still use Funnel from the CLI ([docs](https://tailscale.com/kb/1223/tailscale-funnel/?q=funnel)).

Reviewer: ```suggestion
> Notice for Linux users: We're working to resolve an issue preventing this extension from being used on Linux. However, you can still use Funnel from the CLI ([docs](https://tailscale.com/kb/1223/tailscale-funnel/?q=funnel)).
```
